src/args.py

```diff
@@ -43,6 +58,10 @@
 
 # YOUR CODE HERE
 
+def f4(**kwargs):
+    print(kwargs)
```

Comment:
This code gets you each key value pair one at a time instead of the entire dict. 

`
for k, v in kwargs.items():
  print(f'key: {k}, value: {v}')
`